src/Psalm/Checker/Statements/Expression/CallChecker.php

```diff
@@ -318,11 +318,11 @@ public static function analyzeFunctionCall(
                     $stmt->args,
                     $method_id,
                     $function_params,
+                    $code_location,
```

Comment:
These arguments were shifted around because PSR-2 requires that method arguments with default values go at the end of the list.